src/libraries/Common/src/System/Security/Cryptography/RSAOpenSsl.cs

```diff
@@ -116,11 +114,10 @@ public override byte[] Decrypt(byte[] data, RSAEncryptionPadding padding)
 
             ValidatePadding(padding);
             SafeEvpPKeyHandle key = GetKey();
-            int keySizeBytes = Interop.Crypto.EvpPKeySize(key);
 
-            // OpenSSL requires that the decryption buffer be at least as large as EVP_PKEY_size.
-            // So if the destination is too small, use a temporary buffer so we can match
-            // Windows behavior of succeeding so long as the buffer can hold the final output.
+            // For RSA decryption buffer is equal to key size in bytes
```

Comment:
I think the previous comment was much more descriptive.  Especially since the new one is wrong (the decryption buffer _we pass to OpenSSL_ has to be _at least_ the key size in bytes).

Reply:
note EVP_PKEY_size is not key size in bytes, it's "size which is enough for most of the cryptographic operations" - I will adjust previous comment (therefore why this was adjusted)